docs/html/development/getting-started.rst

```diff
@@ -104,6 +104,15 @@ can select tests using the various ways that pytest provides:
     $ # Using keywords
     $ nox -s test-3.10 -- -k "install and not wheel"
 
+.. note::
+
+    When running pip's tests with OS distribution Python versions, be aware that some
+    functional tests may fail due to potential Pip patches introduced by the distribution.
```

Comment:
```suggestion
    functional tests may fail due to potential patches introduced by the distribution.
```

The patches are usually in Python itself. Let's not discuss what has been patched to avoid nuances like that!

Reply:
Agreed! I was uncomfortable with this statement but couldn't put my finger on why.